### DISCUSSION TITLE
Search Sub-Objects

### DISCUSSION THREAD

**Discussion by weizhongzou** [C-Accel-CRIPT/Python-SDK#180] [Q&A]
For the search function in SDK, can it handle sub-objects in the data model?

**Comment by nh916** [comment]
For search, I believe the API can handle both nodes and sub-objects and I think we can work with that with our SDK API client search method, but I am not entirely certain and don't remember trying it. Give it a try, see what you find, and let me know if there are any issues or possible updates that you see.

For the API, @brili has worked more on it and might have better answers on this.
